examples/owin/Controllers/TestController.cs

```diff
@@ -14,7 +14,6 @@
 // limitations under the License.
 // </copyright>
 
-using System;
```

Comment:
please remove any changes unrelated to this PR's topic.

Reply:
This was hit by my VS `fix for entire solution`, I will revert all of these.